### PR TITLE
Fixture model

### PR DESCRIPTION
**Pull request by javoha**
# Description
Added the model I used for my poster as a possible fixture model. 

Relates to #319 

# Checklist
- [ ] My code follows the [PEP 8 style guidelines](https://www.python.org/dev/peps/pep-0008/).
- [ ] My code uses type hinting for function and method arguments and return values.
- [ ] My code contains descriptive and helpful docstrings 
which are formatted per the [Google Python Style Guidelines](http://google.github.io/styleguide/pyguide.html).
- [ ] I have created tests which entirely cover my code.
- [x] The test code either 1. demonstrates at least one valuable use case (e.g. integration tests) 
or 2. verifies that outputs are as expected for given inputs (e.g. unit tests).
- [x] New and existing tests pass locally with my changes.

### CODE REVIEW

File: test/test_core/test_integration.py [hunk 1]
```diff
@@ -93,14 +93,18 @@ def test_kriging_mutation(interpolator_islith_isfault, map_sequential_pile):
     #plt.savefig(os.path.dirname(__file__)+'/figs/test_kriging_mutation')
 
     geo_model.modify_kriging_parameters('range', 1)
-
     geo_model.modify_kriging_parameters('drift equations', [0, 3])
+
     print(geo_model.solutions.lith_block, geo_model.additional_data)
+    # copy dataframe before interpolator is calculated
+    pre = geo_model.additional_data.kriging_data.df.copy()
```
Comment (pytzcarraldo) [inline test/test_core/test_integration.py]: Hey @javoha! I was wondering what this part of the pull request is about?

Comment (pytzcarraldo) [reply test/test_core/test_integration.py]: Ah, I just saw the commit form a couple of weeks ago on "When using modify_kriging_parameters and recompiling the interpolator, the changes are not taken into account and reset to default." - Thanks!

Comment (javoha) [reply test/test_core/test_integration.py]: Actually I was also wondering - Miguel merged that piece of code, I guess and I kept my remote branch up to date. Still in this pull request it was included - can you tell me why?

Comment (pytzcarraldo) [reply test/test_core/test_integration.py]: Well, apparently it's not part of the master branch - but I can't tell you why 😄 Some merging accident maybe?

Comment (javoha) [reply test/test_core/test_integration.py]: Mmh I think @Leguark wanted to change it a little. Maybe this should not be in this pull request then.

Comment (Leguark) [reply test/test_core/test_integration.py]: yes there is a merge conflict, I will try to deal with it as soon as my enviroment is ready

**Comment by pytzcarraldo** [comment]
@javoha Some cells in `notebooks/examples/ImplicitBenchmark/Claudius.ipynb` are failing with Travis 🤔

**Comment by javoha** [comment]
Mmh just for the fixture models that doesnt make sense - I just added the one notebook and the source files ... Is travis more specific ?

**Comment by pytzcarraldo** [comment]
yeah! by clicking on `Details` next to the Travis build fail you can see the entire log of the test run - and the full error traceback

**Comment by Leguark** [comment]
it should be fixed in the master. @javoha can you pull the master, resolve the conflict and push again?

**Comment by javoha** [comment]
Will do!

**Pull request merged**